scikit/notebooks/Scikit Learn on CMLE.ipynb

```diff
@@ -0,0 +1,381 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Predicting Income with the Census Income Dataset Using Scikit Learn on ML Engine\n",
+    "To use Scikit Learn on ML Engine for predictions, you will need to have a pre-trained model that you can upload to ML Engine. This datalab use the [Census Income Data Set](https://archive.ics.uci.edu/ml/datasets/Census+Income) to create a simple model, train the model, upload the model to ML Engine, and lastly use the model to make predictions. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part 0: Setup\n",
+    "* [Create a project on GCP](https://cloud.google.com/resource-manager/docs/creating-managing-projects)\n",
+    "* [Create a Google Cloud Storage Bucket](https://cloud.google.com/storage/docs/quickstart-console)\n",
+    "* [Enable Cloud Machine Learning Engine and Compute Engine APIs](https://console.cloud.google.com/flows/enableapi?apiid=ml.googleapis.com,compute_component&_ga=2.217405014.1312742076.1516128282-1417583630.1516128282)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download the data\n",
+    "The [Census Income Data Set](https://archive.ics.uci.edu/ml/datasets/Census+Income) that this sample\n",
+    "uses for training is hosted by the [UC Irvine Machine Learning\n",
+    "Repository](https://archive.ics.uci.edu/ml/datasets/). We have hosted the data\n",
+    "on Google Cloud Storage in a slightly cleaned form:\n",
+    "\n",
+    " * Training file is `adult.data.csv`\n",
+    " * Evaluation file is `adult.test.csv`\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Disclaimer\n",
+    "The source of this dataset is from a third party. Google provides no representation,\n",
```

Comment:
Grammar nit: This dataset is provided by a third party.

Reply:
fixed (We should fix the census sample too)

Reply:
👍 Please feel free to send PR :)

Reply:
Will do (waiting for temp project to get whitelisted to show all cells)